### PR TITLE
Fix cmd client

### PR DESCRIPTION
**Pull request by TheOneRing**
Depends on https://github.com/owncloud/client/pull/10149

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests</strong> https://drone.owncloud.com/owncloud/client/13271/6/1

:boom: The GUI tests failed.

GUI Logs: https://cache.owncloud.com/public/owncloud/client/13271/guiReportUpload/index.html

<details>

Server Logs: https://cache.owncloud.com/public/owncloud/client/13271/guiReportUpload/serverlog.log

Screenshots:
- [Filenames_that_are_rejected_by_the_server_are_reported.png](https://cache.owncloud.com/public/owncloud/client/13271/guiReportUpload/screenshots/Filenames_that_are_rejected_by_the_server_are_reported.png)

</details>

File: src/cmd/cmd.cpp [hunk 1]
```diff
@@ -479,9 +412,12 @@ int main(int argc, char **argv)
             std::getline(std::cin, s);
             ctx.user = QString::fromStdString(s);
         }
+#if 0
```
Comment (erikjv) [inline src/cmd/cmd.cpp]: Hmm, why not remove this altogether?

**Review by erikjv — APPROVED** [review]

File: src/cmd/cmd.cpp (hunk 1 above)
Comment (TheOneRing) [reply src/cmd/cmd.cpp]: wait, thats not meant to be still around 🙈

Comment (TheOneRing) [reply src/cmd/cmd.cpp]: Did you look on the single commits?
I don't think that was in the final revision, I had to temporarily disable that code before I finished the refactoring in the last commit.

Comment (erikjv) [reply src/cmd/cmd.cpp]: I only looked at the separate commits, but apparently missed the final removal.

**Pull request merged**